students/mkdir01/lesson08/circle.py

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/python
+
+from math import pi
+
+
+class Circle():
+
+    def __init__(self, radius):
+        self._radius = radius
+
+    @property
+    def radius(self):
+        return self._radius
```

Comment:
No need to use a `@property` to define an instance attribute if you aren't adding any extra logic or calculations. It's fine to pass set `self.radius` directly via the `__init__` function

Reply:
regarding your comment below on this..

step 1 didn't specifically call out making radius to be a property so not required but yes having "protection" on it is best in this case.

however I would recommend using double leading underscores which makes the var "private"; consider below example:
```
>>> class Example:
...     def __init__(self, radius):
...             self._radius = radius
...     @property
...     def radius(self):
...             return self._radius
...
...
>>>
>>> ex = Example(2)
>>> ex._radius = 44
>>> ex.radius
44
>>> class Example:
...     def __init__(self, radius):
...             self.__radius = radius
...     @property
...     def radius(self):
...             return self.__radius
...
>>> ex = Example(2)
>>> ex.__radius = 44
>>> ex.radius
2
```

Reply:
@mkdir-washington-edu if you haven't already watched it this optional video from the course materials this is a great [talk](https://youtu.be/HTLu2DFOdTg) on private versus public attribute access, and how to use getters and setters in python